src/Sulu/Bundle/ContentBundle/Behat/BaseStructureContext.php

```diff
@@ -143,6 +144,7 @@ protected function getContentMapper()
     protected function createStructureTemplate($type, $name, $template)
     {
         $paths = $this->getContainer()->getParameter('sulu.content.structure.paths');
+        // FIX THIS
```

Comment:
Does this comment then have to be here? It's not really verbose, and if you create the fix immediately we probably don't need it.